bool negation
string fluent
string argument1
string argument2
int16 timestep
